Tools/WebKitBot/src/WebKitBot.mjs

```diff
@@ -471,7 +471,12 @@ Type \`help COMMAND\` for help on my individual commands.`,
         let results;
         try {
             const webkitPatchPath = path.resolve("BotWebKit", "Tools", "Scripts", "webkit-patch");
-            results = await execFileAsync(webkitPatchPath, [
+            var pythonPath = which.sync('python3', { nothrow: true })
+            if (!pythonPath) {
```

Comment:
I don't think that we need fallback here - it worked fine without a fallback before (via `/usr/bin/env` inside the script).

Reply:
If I don't have a fallback, would execFileAsync still work with an empty pythonPath? And even if it does, the Python sys.executable would end up with an empty string and break Autoinstall, wouldn't it?

Reply:
Probably best to also remove `nothrow: true`, and have it fail. But it won't fail in this configuration, because we know that Python is in the path.

Having a manual fallback is dangerous, because it could introduce mysterious behavior if something changes. For a service like this, it's better to fail quickly and loudly in this case.